tests/load/locustfiles/load.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Load test shape module."""
+import math
+from typing import Callable, Type
+
+import numpy
+from locust import LoadTestShape, User
+from locustfile import AutopushUser
+
+TickTuple = tuple[int, float, list[Type[User]] | None]
```

Comment:
```suggestion
TickTuple = tuple[int, int, list[Type[User]]]
```

Reply:
I don't think we need the Option here as tick returns this type or None?

Reply:
For reference this is the abstract method definition:
<img width="839" alt="image" src="https://github.com/mozilla-services/autopush-rs/assets/3422688/dcf091c3-3738-4095-a0de-ddfd660b9792">

I think I'll remove the optional as you suggest, but change the `spawn_rate` to a float type to be more consistent with the base. What do you think?